### PR TITLE
IPD 48 Improving Illumos on IPv6-primary and IPv6-only networks

### PR DESCRIPTION
**Pull request by Bill-Sommerfeld**

### CODE REVIEW

File: ipd/0048/README.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+== DNS resolver autoconfiguration
+
+=== DNS configuration via DHCPv6
+
+There is currently no way for DNS configuration to make its way from
+the DHCP client to /etc/resolv.conf or other resolver configuration.
+
+=== DNS configuration from Router Advertisement options
+
+As with DHCPv4, there is currently no way for DNS configuration to
```
Comment (sjorge) [inline ipd/0048/README.adoc]: Should this read DHCpv6 here?

Comment (Bill-Sommerfeld) [reply ipd/0048/README.adoc]: Indeed it should!  Fixed.

**Comment by sjorge** [comment]
Some issues I had bookmarked, as I've been trying to properly run Dual-stack since I started using illumos ... that (might be) relevant for proper IPv6 support.

 - https://www.illumos.org/issues/3358 https://github.com/TritonDataCenter/smartos-live/issues/930 mac and ip spoof protection causes issues with IPv6 traffic in various combinations (from memory: mac protection causes issues with SLAAC and addresses that do not embed the MAC-Addr, ip spoofing blocks IPv6 addresses configured via SLAAC and DHCPv6)
 - https://www.illumos.org/issues/7494 somewhere gethostbyaddr_r swallows IPv6 addresses, not sure if this is still an issue. I can't get rm's test case to compile
 - https://www.illumos.org/issues/4033 prefix delegation, already mentioned but there is an old issue for that too
 - https://www.illumos.org/issues/2338 RDNSS support,  also already mentioned but we also have an old issue for that

**Comment by Bill-Sommerfeld** [comment]
> Some issues I had bookmarked, as I've been trying to properly run Dual-stack since I started using illumos ... that (might be) relevant for proper IPv6 support.

Thanks for the list!   I'll add them (and a few more!) in a later revision when I flesh it out further (right now I mainly want to claim an IPD number)

**Comment by rmustacc** [comment]
Hi Bill, I've pulled this in for now so you can claim the number and related. We'll follow up to simplify iteration for you on this.

**Pull request merged**